### ISSUE TITLE
Feature request: Add base path config / environment option

### ISSUE THREAD

**Issue by TilCreator** [helfrichmichael/orcaslicer-novnc#8]
Some (like myself) might want to run orcaslicer-novnc behind a reverse proxy. This often requires having a sub path for each web application. The novnc interface doesn't respect this currently and just redirects to "/view.html?...". It would be nice to have some environment option to set that base path.

I also found this issue on easy-novnc: https://github.com/pgaskin/easy-novnc/issues/14
So it's possible this option won't be available until the next version of easy-novnc

**Comment by helfrichmichael** [comment]
I don't believe that GitHub is actively maintained anymore, but if that's viable I'm open to implementing it.

**Comment by iden83** [comment]
Hi @helfrichmichael ,

We are using the deployment and we are missing option mentioned in feature request a lot.
We are having many instances in our use case. Would be neatly implemented to put your deployment behind revers proxy.
At this point there are few places where the execution have fixed location in "/vnc" we tried to use [ngx_http_sub_module](http://nginx.org/en/docs/http/ngx_http_sub_module.html for this, but it was simply too complex to handle.

I really hope you would implement this feature and we would be able to stop dedicating for each deployment separate hostname...

Please , please, please :)
